Assets/MixedRealityToolkit.Services/InputSystem/Editor/MixedRealityCanvasInspector.cs

```diff
@@ -0,0 +1,342 @@
+
+            if (EditorGUI.EndChangeCheck())
+            {
+                if (pixelPerfect == PixelPerfect.Inherit)
+                {
+                    m_PixelPerfectOverride.boolValue = false;
+                }
+                else if (pixelPerfect == PixelPerfect.Off)
+                {
+                    m_PixelPerfectOverride.boolValue = true;
+                    m_PixelPerfect.boolValue = false;
+                }
+                else
+                {
+                    m_PixelPerfectOverride.boolValue = true;
+                    m_PixelPerfect.boolValue = true;
+                }
+            }
+
+            EditorGUILayout.PropertyField(m_OverrideSorting);
+            m_SortingOverride.target = m_OverrideSorting.boolValue;
+
+            if (EditorGUILayout.BeginFadeGroup(m_SortingOverride.faded))
+            {
+                GUIContent sortingOrderStyle = null;
+                if (m_AllOverlay)
+                {
+                    sortingOrderStyle = Styles.sortingOrder;
+                }
+                else if (m_NoneOverlay)
+                {
+                    sortingOrderStyle = Styles.m_SortingOrderStyle;
+                    sortingLayerField.Invoke(null, new object[] { Styles.m_SortingLayerStyle, m_SortingLayerID, EditorStyles.popup });
+                }
+                if (sortingOrderStyle != null)
+                {
+                    EditorGUILayout.PropertyField(m_SortingOrder, sortingOrderStyle);
+                }
+            }
+            EditorGUILayout.EndFadeGroup();
+        }
+
+        private bool ShowMRTKButton()
+        {
+            Canvas canvas = (Canvas)target;
+
+            if (!canvas.rootCanvas)
+            {
+                return false;
+            }
+
+            bool isMRTKCanvas = canvas.GetComponent<CanvasUtility>() != null;
+
+            if (isMRTKCanvas)
+            {
+                if (GUILayout.Button("Revert MRTK canvas"))
+                {
+                    EditorApplication.delayCall += () =>
+                    {
+                        DestroyImmediate(canvas.GetComponent<NearInteractionTouchableUnityUI>());
+                        DestroyImmediate(canvas.GetComponent<CanvasUtility>());
+                    };
+
+                    isMRTKCanvas = false;
+                }
+
+                if (canvas.GetComponentInChildren<NearInteractionTouchableUnityUI>() == null)
+                {
+                    EditorGUILayout.HelpBox($"Canvas does not contain any {typeof(NearInteractionTouchableUnityUI).Name} components for supporting near interaction.", MessageType.Warning);
+                    if (GUILayout.Button($"Add {typeof(NearInteractionTouchableUnityUI).Name}"))
+                    {
+                        Undo.AddComponent<NearInteractionTouchableUnityUI>(canvas.gameObject);
+                    }
+                }
+            }
+            else
+            {
+                if (GUILayout.Button("Make MRTK canvas"))
```

Comment:
Can we add some tooltip or info msg here as well that explains what's going to happen or link to documentation, just so it's more clear what's going on.